### PR TITLE
fix: position executor in transaction stepper

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

Resolves incorrect placement of executor in transaction stepper:

image

## How this PR fixes it

The stepper "line" (`::before`) length is relevant to the display of executor/hint about execution.

## How to test it

- Queue a transaction and observe the hint regarding execution positioned correctly.
- Execute a transaction and observe the executor positions correctly.
- Observe no line that spreads beyond the final step.

## Screenshots

image

image

image

image

### CODE REVIEW

File: src/components/transactions/TxSigners/index.tsx [hunk 1]
```diff
@@ -127,10 +129,18 @@ export const TxSigners = ({ txDetails, txSummary }: TxSignersProps): ReactElemen
...
+  const showExecutor = executor || !isConfirmed
 
   return (
     <>
-      <List className={css.signers}>
+      <List
+        className={css.signers}
+        sx={{
+          '::before': {
+            height: `calc(100% - ${showExecutor ? '80px' : '40px'})`,
```
Comment (katspaugh) [inline src/components/transactions/TxSigners/index.tsx:140]: Why so complicated? Can't the executor 🪓 just go directly after everything above?

Comment (iamacook) [reply src/components/transactions/TxSigners/index.tsx:140]: If we don't have conditional `height`, the stepper line bleeds into the executor positioning:

image

vs.

image

Comment (iamacook) [reply src/components/transactions/TxSigners/index.tsx:140]: 🤦‍♂️ I see what you mean now. Let me adjust it.

**Review by katspaugh — APPROVED** [review]

**Comment by katspaugh** [comment]
Tested, it's fixed now 👍

**Pull request merged**